content/en/Getting started/planning.md

```diff
@@ -18,22 +20,132 @@ we can help you schedule pentests with a start date from at least one to three b
+
+The characteristics differ for each [asset type](../assets/asset-type/):
+
+- [Web](#web)
+- [Mobile](#mobile)
+- [API](#api)
+- [External Network](#external-network)
+- [Internal Network](#internal-network)
+- [Cloud Config](#cloud-configuration)
+- [Assets of multiple types](#assets-of-multiple-types)
+
+Once you’ve scoped the pentest, see how many [credits](#view-required-credits) it requires.
+
+### Web
+
+To scope a pentest for a Web asset, specify the number of the following characteristics of that asset:
+
+- [User Roles](../glossary/#user-role)
+- [Pages](../glossary/#page)/[Routes](../glossary/#route)
+
+Include those User Roles and Pages/Routes that should be in scope for this pentest. Be thorough. If you forget certain roles or pages/routes, your pentest might not cover all critical details.
+
+{{% alert title="Note" color="note" %}}
+You may not need to include _every_ user role. For example, if you have dedicated administrative roles for backups, logs, and printers, that counts as one (1) role.
+{{% /alert %}}
+
+As part of our tests for dynamic pages, we also test the backend API endpoints frequently used to populate content on those pages.
+
+Our pentesters need to know more about your Web asset, including:
+
+- Application type, such as a page-driven website or a [single-page application](../glossary/#single-page-application)
+- Special endpoints associated with your pages
+
+{{% alert title="Note" color="note" %}}
+If the only APIs in your assets populate dynamic web pages, you may not need to set up a separate API asset. We test such APIs as part of our tests of a Web asset.
+{{% /alert %}}
+
+### Mobile
+
+To scope a pentest for a Mobile asset, specify the number of the following characteristics of that asset:
+
+- [User Roles](../glossary/#user-role)
+- [Screens](../glossary/#mobile-screen) (for all OS)
+
+Include those User Roles and Screens that should be in scope for this pentest. Be thorough. If you forget certain roles or screens, your pentest might not cover all critical details.
+
+### API
+
+We can test both RESTful and GraphQL APIs. However, these APIs work in different ways. While some RESTful APIs can have dozens of endpoints, a GraphQL API has a single endpoint.
+
+If you’re sizing a GraphQL API, identify a list of [queries and mutations](https://graphql.org/learn/queries). For pentest purposes, that’s functionally equivalent to the number of RESTful API endpoints.
+
+To scope a pentest for an API asset, specify the number of the following characteristics of that asset:
+
+- [User Roles](../glossary/#user-role)
+- [Endpoints](../glossary/#api-endpoint) / [GraphQL Mutations](../glossary/#graphql-mutation)
+
+Include those User Roles and Endpoints / GraphQL Mutations that should be in scope for this pentest. If you forget some, you may sacrifice quality in penetration testing.
```

Comment:
I'm realizing that I should have specified a `RESTful API Endpoint` in the Glossary. For our purposes, it's the same category as a `GraphQL Mutation`. 

Short explanation (IIRC): RESTful APIs set up data on different endpoints. GraphQL uses a single endpoint, but uses mutations to manage different categories of data.